pages/index.tsx

```diff
@@ -1,71 +1,110 @@
 import type { NextPage } from "next";
-import Head from "next/head";
-import Image from "next/image";
-import styles from "../styles/Home.module.css";
+// import dynamic from "next/dynamic";
+import classnames from "classnames";
 
-const Home: NextPage = () => {
-  return (
-    <div className={styles.container}>
-      <Head>
-        <title>Create Next App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-
-      <main className={styles.main}>
-        <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
-        </h1>
-
-        <p className={styles.description}>
-          Get started by editing{" "}
-          <code className={styles.code}>pages/index.js</code>
-        </p>
+import Button from "../components/atoms/Button/Button";
+import { SubTitle } from "../components/atoms/SectionTitle/SectionTitle";
+import Card from "../components/atoms/Card/Card";
+import CollapseBox from "../components/atoms/CollapseBox/CollapseBox";
+import SectionWrapper from "../components/molecules/SectionWrapper/SectionWrapper";
+import BannerContainer from "../components/molecules/BannerContainer/BannerContainer";
```

Comment:
Add `index.tsx` for each component to make clear for exporter and make easier for imoprter.

```suggestion
import Button from "../components/atoms/Button";
import { SubTitle } from "../components/atoms/SectionTitle";
import Card from "../components/atoms/Card";
import CollapseBox from "../components/atoms/CollapseBox";
import SectionWrapper from "../components/molecules/SectionWrapper";
import BannerContainer from "../components/molecules/BannerContainer";
```